orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTaskSpec.groovy

```diff
@@ -146,15 +146,65 @@ class MonitorKatoTaskSpec extends Specification {
     "about to fail"      | MonitorKatoTask.TASK_NOT_FOUND_TIMEOUT     |  true         | 'N/A'
   }
 
+  def "should set kato.task.skipReplica=true when getting a task from clouddriver times out"() {
+    given:
+    def taskId = "katoTaskId"
+    def elapsed = MonitorKatoTask.TASK_NOT_FOUND_TIMEOUT + 1
+    task.kato = Mock(KatoService)
+
+    def stage = new Stage(Execution.newPipeline("orca"), "whatever", [
```

Comment:
There is a shorthand way of doing I think ..

```
def stage = stage {
  context = [
    "kato.last.task.id": ...
  ]
}
```

`stage {` comes from ExecutionBuilder